internal/config/config.go

```diff
@@ -264,25 +264,26 @@ func InitConfig(override, defaultConfig string) {
 
 	configData.defaultConfig, configData.AppConfig, err = readViperConfig(appName, configDirs)
 	if err != nil {
-		err = os.MkdirAll("config", os.ModePerm)
-		if err != nil {
-			log.Fatal("unable to create configuration folder: 'config'")
+		loadDefault := tools.GetUserConfirmation("unable to load configuration.  Would you like to fall back on default? This will overwrite the current configuration.  (y/n) ", "", true)
```

Comment:
Hm is there a way to separate between "no config found" vs "bad config" here? I may not be understanding the full code flow here. "No config found" would feel like a good time to generate a default config, but bad config feels like we should bail.

Reply:
Fixed